src/plugins/vencordToolbox/index.tsx

```diff
@@ -19,59 +19,267 @@
+                note="Display Relaunch Discord entry in toolbox"
+            >
+                Relaunch Discord
+            </Switch>
+    },
+    Notifications: {
+        type: OptionType.COMPONENT,
+        description: "Open Notification Log",
+        default: true,
+        component: () =>
+            <Switch
+                value={settings.store.Notifications}
+                onChange={(v: boolean) => settings.store.Notifications = v}
+                note="Display Notifications Log entry in toolbox"
+            >
+                Notification Log
+            </Switch>
+    },
+    QuickCSS: {
+        type: OptionType.COMPONENT,
+        description: "Open QuickCSS",
+        default: true,
+        component: () =>
+            <Switch
+                value={settings.store.QuickCSS}
+                onChange={(v: boolean) => settings.store.QuickCSS = v}
+                note="Display QuickCSS entry in toolbox"
+            >
+                QuickCSS
+            </Switch>
+    },
+    disableQuickCSS: {
+        type: OptionType.COMPONENT,
+        description: "Toggle QuickCSS",
+        default: true,
+        component: () =>
+            <Switch
+                value={settings.store.disableQuickCSS}
+                onChange={(v: boolean) => settings.store.disableQuickCSS = v}
+                note="Display quick enable/disable QuickCSS in toolbox"
+            >
+                Enable/Disable QuickCSS toggle
+            </Switch>
+    },
+    UpdaterTab: {
+        type: OptionType.COMPONENT,
+        description: "Add Updater",
+        default: false,
+        component: () =>
+            <Switch
+                value={settings.store.UpdaterTab}
+                onChange={(v: boolean) => settings.store.UpdaterTab = v}
+                disabled={IS_WEB}
+                note="Display UpdaterTab entry in toolbox"
+            >
+                UpdaterTab
+            </Switch>
+    },
+
+    // for enabling and disabling misc plugin quick actions
+    BadgeAPI: {
+        type: OptionType.COMPONENT,
+        description: "BadgeAPI settings",
+        default: true,
+        component: () =>
+            <Switch
+                value={settings.store.BadgeAPI}
+                onChange={(v: boolean) => settings.store.BadgeAPI = v}
+                note="Display BadgeAPI entry in toolbox"
+            >
+                BadgeAPI
+            </Switch>
+    },
+    DevCompanion: {
+        type: OptionType.COMPONENT,
+        description: "DevCompanion settings",
+        default: true,
+        component: () =>
+            <Switch
+                value={settings.store.DevCompanion}
+                onChange={(v: boolean) => settings.store.DevCompanion = v}
+                note="Display DevCompanion Reconnect entry in toolbox"
+            >
+                DevCompanion
+            </Switch>
+    },
+    // For enabling and disabling individual plugin settings menus
+    PluginSettings: {
```

Comment:
camelCase is preferred
```suggestion
    pluginSettings: {
```

Reply:
okay this one is explainable, before adding the checkbox dropdown menu I had set the settings name to be the name of the plugin to distinguish which plugin to add to the toolbox. but that's all been rendered obsolete ✅